modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/js/info_box/modal/InfoBoxModalNotesInput.js

```diff
@@ -0,0 +1,168 @@
+import {ClayInput, ClayToggle} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import {InfiniteScrollerComponent} from 'commerce-frontend-js';
+import moment from 'moment';
+import React from 'react';
+
+import UserIcon from '../../UserIcon';
+
+const InfoBoxModalNotes = ({
+	handleDelete,
+	handleToggle,
+	isRestricted,
+	notes,
+	setInputValue,
+	spritemap,
+}) => {
+	const formatUTCDate = (value) => {
+		return moment
+			.utc(value)
+			.locale(Liferay.ThemeDisplay.getBCP47LanguageId())
+			.startOf('seconds')
+			.fromNow();
+	};
+
+	return (
+		<>
+			<InfiniteScrollerComponent maxHeight="300px" scrollCompleted>
+				<div className="lfr-discussion-body">
+					{notes.map((note, index) => {
+						const {
+							author,
+							authorId,
+							authorPortraitURL,
+							content,
+							id,
+							modifiedDate,
+							restricted,
+						} = note;
+
+						return notes ? (
+							<div className="panel-body">
+								<div className="card-row d-flex mb-2">
+									<UserIcon
+										fullName={author}
+										portraitURL={authorPortraitURL}
+										userId={authorId}
+									/>
+
+									<div
+										className="flex-grow-1 pl-2"
+										key={index}
+									>
+										<div className="d-flex mb-1">
+											<span className="author">
+												{author}
+
+												{themeDisplay.getUserId() ===
+												authorId.toString()
+													? ` (${Liferay.Language.get('you')})`
+													: null}
+											</span>
+
+											<span className="d-flex m-1 small">
+												{formatUTCDate(modifiedDate)}
+											</span>
+										</div>
+
+										<div>
+											{restricted ? (
+												<ClayIcon
+													spritemap={spritemap}
+													symbol="lock"
+												/>
+											) : null}
+										</div>
+
+										<div className="d-flex mb-2">
+											{content}
+										</div>
+									</div>
+
+									<div className="card-col">
+										<ClayDropDown
+											trigger={
+												<ClayButtonWithIcon
+													aria-label={Liferay.Language.get(
+														'more-actions'
+													)}
+													className="btn-outline-borderless btn-outline-secondary"
+													displayType="unstyled"
+													symbol="ellipsis-v"
+												/>
+											}
+										>
+											<ClayDropDown.ItemList>
+												<ClayDropDown.Item
+													onClick={() => {
+														handleDelete(id);
+													}}
+												>
+													{Liferay.Language.get(
+														'delete'
+													)}
+												</ClayDropDown.Item>
+											</ClayDropDown.ItemList>
+										</ClayDropDown>
+									</div>
+								</div>
+
+								<hr className="mt-1 separator" />
+							</div>
+						) : null;
+					})}
+				</div>
+			</InfiniteScrollerComponent>
+
+			<ClayInput.Group className="commerce-panel">
+				<ClayInput.GroupItem>
+					<ClayInput
+						aria-required={true}
+						className="field form-control lfr-textarea"
```

Comment:
@gianmarcobrunialti please remember to add the missing css class to this text area